pkg/servicecentral/types.go

```diff
@@ -35,6 +34,13 @@ type ServiceData struct {
 	Compliance *ServiceComplianceConf `json:"compliance,omitempty"`
 }
 
+type ServiceDataRead struct {
```

Comment:
Compliance is also read-only, should move that out of Write.

Reply:
OpsGenie Team is readonly too /cc @fcobb